### ISSUE TITLE
@!visibility YARD metadata tag support

### ISSUE THREAD

**Issue by nfedyashev** [castwide/solargraph#566]
Just wanted to clarify if @!visibility meta tags are not supported(yet) or is it something with my local setup?

The following class always autosuggests both methods in autocompletion and "Visibility: public" is displayed in popup for both methods.

```
class Foo
  # @!visibility private
  def bar
  end

  # @!visibility protected
  def baz
  end
end
```

I know `private` method modifier works great in Solagraph and I probably just use that.

The original idea was just to change visibility to private(and exclude from autocompletion) a method that is rarely used in the project but still need to be public.

Solargraph version: `master` branch

**Comment by castwide** [comment]
Solargraph doesn't support the `@!visibility` directive yet, but it should be easy to implement. I'll see if I can get it into the next release.

**Comment by nfedyashev** [comment]
@castwide thank you very much!

**Comment by castwide** [comment]
Released in v0.47.0.
